docs/v3.x/concepts/models.md

```diff
@@ -164,12 +164,18 @@ The options key on the model-json states.
 
 - `timestamps`: This tells the model which attributes to use for timestamps. Accepts either `boolean` or `Array` of strings where first element is create date and second element is update date. Default value when set to `true` for Bookshelf is `["created_at", "updated_at"]` and for MongoDB is `["createdAt", "updatedAt"]`.
 
+- `privateAttributes`: This configuration allows to treat a set of attributes as private, even if they're not actually defined as attributes in the model. Accepts an `Array` of strings. It could be used to remove from API responses timestamps or `_v` when using MongoDB. The set of `privateAttributes` defined in the model are merged with the `privateAttributes` defined in the global Strapi configuration.
+
+- `ignorePrivateFor`: This configuration allows to ignore some attributes from the global `privateAttributes` configuration. Accepts an `Array` of strings. The attributes defined in this `Array` will be removed from the final merged `Array` from the model and global `privateAttributes` configuration.
```

Comment:
Shouldn't this be "from the global **or local** `privateAttributes` configuration"?